app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -68,7 +68,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
         sessionManager.handleIntent(this, intent, savedInstanceState);
 
-        sessionManager.getSessions().observe(this,  new NonNullObserver<List<Session>>() {
+        registerSessionObserver();
+
+        WebViewProvider.preload(this);
+    }
+
+    private void registerSessionObserver() {
```

Comment:
\[PMD\] INFO: The method 'registerSessionObserver' has a Cyclomatic Complexity of 6\.